docs/content/en/docs/pipeline/chain-security.md

```diff
@@ -0,0 +1,370 @@
+This file will be used to create the required Kubernetes secrets for image repository authentication.
+
+#### Creating Secrets for Task Image Repository Authentication
+
+Create a secret for tasks to upload images to the OCI repository:
+
+```console
+kubectl create secret generic docker-credentials --from-file=/root/.docker/config.json -n kurator-pipeline
+```
+
+This secret, used as a workspace parameter in tasks, grants authentication access.
+
+#### Creating Secrets for Chain Controller
+
+Create a secret for the chain controller to upload signatures(sig) and attestation (att) to the OCI repository:
+
+```console
+kubectl create secret generic chain-credentials \
+    --from-file=.dockerconfigjson=/root/.docker/config.json \
+    --type=kubernetes.io/dockerconfigjson \
+    -n kurator-pipeline
+```
+
+### Configuring Tekton Chains Parameters
+
+To ensure proper functioning of Tekton Chains components, apply the following configurations:
+
+```console
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.taskrun.format": "slsa/v1"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.taskrun.storage": "oci"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.oci.storage": "oci"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"transparency.enabled": "true"}}'
+```
+
+With these steps completed, the preliminary setup is ready, paving the way for secure and efficient pipeline operations for supply chain security.
+
+## Triggering the Test Pipeline
+
+### Creating Kurator Pipeline Test Example
+
+Create a pipeline that includes an image task using the following command:
+
+```console
+echo 'apiVersion: pipeline.kurator.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: quick-start
+  namespace: kurator-pipeline
+spec:
+  description: "this is a quick-start pipeline, it shows how to use customTask and predefined Task in a pipeline"
+  tasks:
+    - name: git-clone
+      predefinedTask:
+        name: git-clone
+        params:
+          git-secret-name: git-credentials
+    - name: cat-readme
+      customTask:
+        image: zshusers/zsh:4.3.15
+        command:
+          - /bin/sh
+          - -c
+        args:
+          - "cat $(workspaces.source.path)/README.md"
+    - name: go-test
+      predefinedTask:
+        name: go-test
+        params:
+          packages: ./...
+    - name: go-lint
+      predefinedTask:
+        name: go-lint
+        params:
+          packages: "./..."
+          flags: "--disable=errcheck,unused,gosimple,staticcheck --verbose --timeout 10m"
+    - name: build-and-push-image
+      predefinedTask:
+        name: build-and-push-image
+        params:
+          image: "ghcr.io/<username>/<image-name>"'| kubectl apply -f -
+```
+
+Replace `<username>` with your GitHub username and `<image-name>` with your image name, like `kurator-test:0.4.1`.
+
+### Exposing Services
```

Comment:
Should be clear this service if for webhook, but not every one know that

Reply:
in last  doc, I already explain it, may be we can add link for it.